packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -462,12 +466,97 @@ class DebuggerController extends DisposableController
   void selectStackFrame(StackFrameAndSourcePosition frame) {
     _selectedStackFrame.value = frame;
 
+    if (frame?.frame != null) {
```

Comment:
Is is possible for the `frame` field on a `StackFrameAndSourcePosition` instance to be null?

Reply:
good question. changed to `if (frame != null)`